cdc/puller/frontier/frontier.go

```diff
@@ -163,3 +164,29 @@ func (s *spanFrontier) String() string {
 	})
 	return buf.String()
 }
+
+// SpanString returns the string of the span's frontier.
+func (s *spanFrontier) SpanString(span tablepb.Span) string {
+	var buf strings.Builder
+	idx := 0
+	s.spanList.Entries(func(n *skipListNode) bool {
+		key := n.Key()
+		nextKey := []byte{}
+		if n.Next() != nil {
```

Comment:
I'm not sure whether it's correct or not. Maybe call `s.Entries` is better.

Reply:
We need to access some inner fileds of skipListNode, but `s.Entries` passes only key and value.